### ISSUE TITLE
Reading incorrect color data when JpegColorType is Rgb.

### ISSUE THREAD

**Issue by CryShana** [SixLabors/ImageSharp#2141]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have verified that I am running the latest version of ImageSharp
- [X] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [X] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### ImageSharp version

2.1.2

### Other ImageSharp packages and versions

SixLabors.ImageSharp 2.1.2

### Environment (Operating system, version and so on)

Windows 11 and Ubuntu 20.04.4 LTS

### .NET Framework version

.NET 6 (SDK 6.0.300)

### Description

JPEG images with ColorType value of  `JpegColorType.Rgb` have their color values read/parsed incorrectly. 

Though such images are rare, I encountered multiple such images already and all of them were read incorrectly. Other image readers and tools read them correctly.

Example code:
```cs
using var image = Image.Load(filePath);

var colorType = image.Metadata.GetJpegMetadata().ColorType;
if (colorType == JpegColorType.Rgb)
{
    // colors will be read wrong
}

// you can save it as anything, doesn't matter
image.SaveAsJpeg("output.jpg");
```

Just to get an idea of how colors change with an example input image. This is the original:
![input](https://user-images.githubusercontent.com/951230/172003586-072c5a9f-f692-4932-bd72-33e9ac885c5f.jpg)

The output is this:
![output](https://user-images.githubusercontent.com/951230/172003599-59517c5a-9786-4d77-ad2d-41ca3c1ef22a.jpg)

### Steps to Reproduce

This works on any JPEG image that has `ColorType` = `JpegColorType.Rgb`

- Load any such image by using `Image.Load(filepath);` or `Image.Load<Rgba32>(filepath);`
- You can try reading some pixels to confirm the colors are off
- Save image using any method, I used `image.SaveAsJpeg(outputPath);`

The output should be very different from the input.

### Images

Original (input image):
![input](https://user-images.githubusercontent.com/951230/172002752-b4f315ac-2e1d-4970-a898-314fc2f3bfc6.jpg)

**Comment by brianpopow** [comment]
@CryShana this is probably already fixed with #2135. Can you try our nightly build form the [MyGet feed](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp)?

**Comment by CryShana** [comment]
Ah, you are correct. I just checked out the `3.0.0-alpha.0.11` version of ImageSharp and it works now.

Do I close this issue?
